PyPDF2/generic.py

```diff
@@ -1850,6 +1850,16 @@ def bottom(self) -> Optional[FloatObject]:
         """Read-only property accessing the bottom vertical coordinate."""
         return self.get("/Bottom", None)
 
+    @property
+    def color(self) -> Optional[tuple]:
+        """Read-only property accessing the color in (R, G, B) with values 0.0-1.0"""
+        return self.get("/C", None)
+
+    @property
+    def fmt(self) -> Optional[int]:
```

Comment:
Instead of returning `Optional[int]` you could define:

```
>>> from enum import IntFlag
>>> class OutlineItemFlag(IntFlag):
...     italic = 1
...     bold = 2

>>> OutlineItemFlag.italic | OutlineItemFlag.bold
<OutlineItemFlag.bold|italic: 3>

>>> OutlineItemFlag.italic | OutlineItemFlag.bold == 3
True

>>> OutlineItemFlag(1)
<OutlineItemFlag.italic: 1>

>>> OutlineItemFlag(2)
<OutlineItemFlag.bold: 2>

>>> OutlineItemFlag(3)
<OutlineItemFlag.bold|italic: 3>

>>> OutlineItemFlag.bold == 2
True
```

This is way more convenient to use and more helpful as a return value.

I'm not sure about None, though. Being an int-flag I would have expected 0 as a default.

Reply:
What do you think?

Reply:
Love it; was unfamiliar with that library. 

Similarly: do you think the color should also return the default value rather than `None`? Specifically, `(0.0, 0.0, 0.0)`?

Reply:
Cool :-)

Reply:
Yes, the triplet instead of none sounds better to me